samples/snippets/src/main/java/aiplatform/ImportDataVideoActionRecognitionSample.java

```diff
@@ -68,6 +68,11 @@ static void importDataVideoActionRecognitionSample(
       OperationFuture<ImportDataResponse, ImportDataOperationMetadata> response =
           client.importDataAsync(name, importConfigs);
 
+      while (!response.isDone()) {
+        System.out.println("Waiting for response...");
+        Thread.sleep(10000);
```

Comment:
Please use the `java.time` package instead.

Reply:
at line 81, it is already doing the same thing. No need to add while loop in the sample

https://github.com/googleapis/java-aiplatform/blob/5a0b004dfecf78a6eea6f98ccf7ef559f21440cd/samples/snippets/src/main/java/aiplatform/ImportDataVideoActionRecognitionSample.java#L81